local-dev/render-config.sh

```diff
@@ -9,10 +9,13 @@ VAULT_SERVICE_ACCOUNT_PATH=secret/dsde/terra/kernel/integration/toolsalpha/crl_j
 VAULT_CLIENT_SERVICE_ACCOUNT_PATH=secret/dsde/terra/kernel/integration/toolsalpha/crl_janitor/client-sa
 VAULT_TOOLS_CLIENT_SERVICE_ACCOUNT_PATH=secret/dsde/terra/kernel/integration/tools/crl_janitor/client-sa
 VAULT_CLOUD_ACCESS_SERVICE_ACCOUNT_PATH=secret/dsde/terra/janitor-test/default/cloud-access-sa
+VAULT_AZURE_MANAGED_APP_CLIENT_PATH=secret/dsde/terra/kernel/integration/tools/crl_janitor/azure-managed-app-client
```

Comment:
how did you pull those secrets into vault? 
Can we put that into terraform:
https://github.com/broadinstitute/terraform-ap-modules/blob/master/crl-janitor/outputs.tf
https://github.com/broadinstitute/terraform-ap-deployments/blob/master/terra-env/vault_crl_janitor.tf

Reply:
Also in the same Terraform module, we grant Janitor DELETE permissions on those GCP folders. Can we also have that for Azure.

Reply:
> Can we put that into terraform
Yes! Will make a PR to terraform this. (I wrote a single secret manually but that's probably not best practice :)).

> we grant Janitor DELETE permissions on those GCP folders. Can we also have that for Azure.
Hm, I think the Azure model is slightly different. Janitor will have the identity of a managed app _publisher_, which can delete resources in a managed app, but can't delete the managed app itself. I'm picturing we'll have a static managed app deployment used for integration testing.